src/components/Pages/ManageWiki/Cards/Logo.vue

```diff
@@ -11,6 +11,7 @@
         :persistent-hint="true"
         @change="onLogoFileChanged"
       ></v-file-input>
+      <v-img v-if="logo" :src="logo" max-width="135" />
```

Comment:
Hmm when I'm testing with the suggested hack in https://github.com/wbstack/api/issues/37 I'm still not seeing this however the logo is there so Im guessing the upload just didn't work. Seems to read / write it successfully from db though.